sdk/digitaltwins/Azure.DigitalTwins.Core/api/Azure.DigitalTwins.Core.netstandard2.0.cs

```diff
@@ -156,9 +156,9 @@ public partial class DigitalTwinMetadata
         [System.Text.Json.Serialization.JsonExtensionDataAttribute]
         public System.Collections.Generic.IDictionary<string, object> WriteableProperties { get { throw null; } }
     }
-    public partial class ModelProperties
+    public partial class DigitalTwinsModelProperties
```

Comment:
I don't recall this being one that was renamed to be more specific.

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelProperties.java

Looks like that is what we have in master currently